.github/issue_template.md

```diff
@@ -0,0 +1,27 @@
+### If it's a discussion, please start one [here](https://github.com/cultureamp/kaizen-design-system/discussions).
+
+- [ ] Before opening this issue, I've searched open and closed issues for a fix.
+
+## Expected Behavior
+
+
+## Actual Behavior (please provide screenshots)
+
+
+## Steps to Reproduce the Problem
+
+1.
+2.
+3.
+
+## Specifications (please add any other notable libs/details)
+
+- Kaizen version:
+- Repo:
+
+## Labelling
+
+1. If related to a component, please add a label like so `component:mycomponent`.
+2. Then label with `bug`, `enhancement`/`feature`\* or `documentation`.
+
+**\* A feature is a new component/functionality and an enhancement is an improvement on existing component/functionality.**
```

Comment:
Is functionality a plural? If I read "a new functionality" it sounds wrong. I'm tempted to reword this to avoid that :)

```suggestion
**\* A feature is a new component or piece of functionality and an enhancement is an improvement to an existing component or piece of functionality.**
```

Reply:
Good catch, moved it around and missed that.